ocs_ci/utility/aws.py

```diff
@@ -427,6 +458,159 @@ def get_ec2_instance_volumes(self, instance_id):
         volumes = instance.volumes.all()
         return [vol for vol in volumes]
 
+    def get_security_groups_by_instance_id(self, instance_id):
+        """
+        Get all attached security groups of ec2 instance
+
+        Args:
+            instance_id (str): Required instance to get security groups from it
+
+        Returns:
+            list: all_sg_ids: all attached security groups id.
+
+        """
+        ec2_instance = self.get_ec2_instance(instance_id)
+        all_sg_ids = [sg.get('GroupId') for sg in ec2_instance.security_groups]
+
+        return all_sg_ids
+
+    def create_security_group(self, group_name, dict_permissions, vpc_id):
+        """
+        Create security group with predefined group name and permissions
+
+        Args:
+            group_name (str): Group name (aws tag: "Group Name")
+            dict_permissions (dict): The security group's inbound/outbound permissions
+            vpc_id(str): For group to be attached
+
+        Returns:
+            str: newly created security group id
+
+        """
+        instance_response = self.ec2_client.create_security_group(
+            GroupName=group_name,
+            Description='This group created by method:aws.create_security_group',
+            VpcId=vpc_id
+        )
+
+        security_group_id = instance_response['GroupId']
+        logger.info(f'Security Group Created {security_group_id} in vpc {vpc_id}')
+
+        data = self.ec2_client.authorize_security_group_ingress(
+            GroupId=security_group_id,
+            IpPermissions=[dict_permissions]
+        )
+        logger.info(f'Ingress Successfully Set {data}')
+        return security_group_id
+
+    def append_security_group(self, security_group_id, instance_id):
+        """
+        Append security group to selected ec2 nodes
+
+        Args:
+            instance_id(str): Instances to attach security group
```

Comment:
if you are pushing more commit/s, please fix that 2 lines:
add space between the argument name and type